adapters/improvedigital/improvedigital.go

```diff
@@ -140,6 +140,7 @@ func (a *ImprovedigitalAdapter) MakeBids(internalRequest *openrtb2.BidRequest, e
 	}
 
 	bidResponse := adapters.NewBidderResponseWithBidsCapacity(len(seatBid.Bid))
+	bidResponse.Currency = bidResp.Cur
```

Comment:
Could you please add a json test case for this change?

Reply:
The global code that compares the test bid responses actually doesn't check the currency. Can I add it there? Otherwise, test for responses with different currencies will succeed.

Reply:
@gargcreation1992 Test submitted. As I said above, it would pass also without the adapter fix because the global test logic doesn't check currency in the bid. I propose this fix: https://github.com/prebid/prebid-server/commit/84deddf700786afb49203da7ed183d4f79058a4d
However, fixing the testing logic is out of scope of this PR as it breaks quite a few adapter tests with missing currency and also potentially revealing bugs in some other adapters similar to the one fixed in this PR for Improve Digital adapter.